manifests/params.pp

```diff
@@ -17,21 +17,28 @@
   $version = 'latest'
   $service = 'jenkins'
   $install_from_git = false
+  $install_from_pip = true
+  $install_from_pkg = false
   $git_revision     = 'master'
   $git_url          = 'https://git.openstack.org/openstack-infra/jenkins-job-builder'
 
   case $::osfamily {
     'RedHat', 'Amazon': {
-      $python_packages = [ 'python', 'python-devel', 'python-pip', 'python-argparse']
+      $python_packages = [ 'python', 'python-devel', 'python-pip', 'python-argparse', 'PyYAML']
```

Comment:
Jenkins job builder itself seems to work fine without it - it's possible that it was brought into the python core in the newer version available in CentOS7 .. either way, the module and job builder seem to work without any issues when `python-argparse` is absent